internal/deps/deps.go

```diff
@@ -0,0 +1,146 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+// Package deps provides mechanisms to Node dependencies using docker
+package deps
+
+import (
+	"context"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/cartesi/rollups-node/internal/config"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	DefaultPostgresDockerImage = "postgres:13-alpine"
+	DefaultPostgresPort        = "5432"
+	DefaultPostgresPassword    = "password"
+	DefaultDevnetDockerImage   = "cartesi/rollups-devnet:devel"
+	DefaultDevnetPort          = "8545"
+
+	numPostgresCheckReadyAttempts = 2
+	pollInterval                  = 5 * time.Second
+)
+
+type DepsConfig struct {
+	PostgresDockerImage string
+	PostgresPort        string
+	PostgresPassword    string
+	DevnetDockerImage   string
+	DevnetPort          string
+}
+
+func NewDefaultDepsConfig() *DepsConfig {
+	return &DepsConfig{
+		DefaultPostgresDockerImage,
+		DefaultPostgresPort,
+		DefaultPostgresPassword,
+		DefaultDevnetDockerImage,
+		DefaultDevnetPort,
+	}
+}
+
+// Struct to hold the dependencies containers and control structures
+// to be able to gracefully terminate the containers after they are
+// not needed anymore
+type DepsContainers struct {
+	containers *[]testcontainers.Container
+	waitGroup  *sync.WaitGroup
```

Comment:
```suggestion
	containers []testcontainers.Container
	waitGroup sync.WaitGroup
```

These fields don't need to be pointers.